### DISCUSSION TITLE
Google Wallet integration

### DISCUSSION THREAD

**Discussion by RenatoBorgess** [pretix/pretix#4486] [Ideas] [answered]
Hi team, I'm new to the community and I didn't find any mentions of Google Wallet in the project, however I saw that there's an integration with Apple Wallet. Is it in the road map to integrate Event Tickets from Google Wallet?
https://developers.google.com/wallet/tickets/events

I work supporting Google Wallet and Google Pay APIs and would love to help with any discussions about it.

**Comment by pc-coholic** [comment, marked as answer]
You might have overlooked https://github.com/pretix/pretix/issues/979

Tl;Dr: it will come eventually, but not before "the big wallet rewrite" that I've been pushing away for a long time already...

**Reply by RenatoBorgess** [reply]
Definitely overlooked because it was rebranded from Google Pay Passes to Google Wallet. Sorry about that.
There's a NFC functionality for Event Tickets that I think would be nice to have a look. https://developers.google.com/wallet/smart-tap 

Thanks a lot for the quick response! I'll keep an eye on the wallet rewrite. Is there a discussion about it that I can follow?

**Reply by pc-coholic** [reply]
I aware of that - and SmartTap as well as VAS/ECP will be part of the rewrite...

There is no other discussion thread - feel free to add to it :)
